code/game/jobs/job/upp.dm

```diff
@@ -29,9 +29,9 @@
 	S.update_icon()
 
 	var/obj/item/storage/pouch/bayonet/RS = new /obj/item/storage/pouch/bayonet(H)
-	RS.contents += /obj/item/weapon/combat_knife/upp
-	RS.contents += /obj/item/weapon/combat_knife/upp
-	RS.contents += /obj/item/weapon/combat_knife/upp
+	RS.contents += new /obj/item/weapon/combat_knife/upp
```

Comment:
dont do this, use the proper storage proc `/obj/item/storage/proc/handle_item_insertion()`

Reply:
This would require a larger rework and is out of scope as I'll be doing a whole another system with StonedMC because fuck RoleAuthority.

Reply:
yeah okay